README.md

```diff
@@ -1,22 +1,32 @@
-# Container Escape Bounty
+# Container Escape Bounty CTF
 
-## How to try
+This repository contains config to deploy and install virtual machines for a
+"Container Escape Bounty" Capture the Flag (CTF) game based on [contained.af](https://github.com/genuinetools/contained.af).
 
-Manual preparation:
-- Get a key pair's name from the AWS console. Let's say its name is `testkey`.
+It uses a modified version of contained.af that can be found here: TODO
 
-Prepare `terraform.tfvars` from its template file.
+## Setup
+
+### Requirements
+
+* AWS IAM credentials for programmatic access. Provide them [through environment
+  variables or a shared credentials file](https://www.terraform.io/docs/providers/aws/#authentication)
+  to Terraform.
```

Comment:
could you say which roles should have the IAM credentials: EC2 admin & route53.

Reply:
Updated.